### PR TITLE
Print additional info when id controller not found in NodeIdReuseStressIT

### PR DESCRIPTION
**Pull request by MishaDemianenko**

### CODE REVIEW

File: enterprise/kernel/src/test/java/org/neo4j/kernel/impl/enterprise/store/id/NodeIdReuseStressIT.java [hunk 1]
```diff
@@ -120,7 +121,17 @@ private static void maybeRunIdMaintenance( GraphDatabaseService db, int iteratio
...
-            resolver.resolveDependency( IdController.class ).maintenance();
+            IdController idController = resolver.resolveDependency( IdController.class );
+            if ( idController != null )
+            {
+                idController.maintenance();
+            }
+            else
+            {
+                System.out.println( "Id controller is null. Dumping resolver content." );
+                System.out.println( "Resolver: " + ReflectionToStringBuilder.toString( resolver ) );
```
Comment (burqen) [inline enterprise/kernel/src/test/java/org/neo4j/kernel/impl/enterprise/store/id/NodeIdReuseStressIT.java:132]: I tried and you will fill the log with lines like this for every single page in the page cache:
```
Page[ id = 134, address = 0, filePageId = -1, swapperId = 0, usageCounter = 0 ] OffHeapPageLock[Flush: 0, Excl: 1, Mod: 0, Ws: 0, S: 0]
```
Maybe filter those out, or if you are fine with that then let's go ahead. I'll put "require change".

**Review by burqen — CHANGES_REQUESTED** [review]
See comment

File: enterprise/kernel/src/test/java/org/neo4j/kernel/impl/enterprise/store/id/NodeIdReuseStressIT.java (hunk 1 above)
Comment (MishaDemianenko) [reply enterprise/kernel/src/test/java/org/neo4j/kernel/impl/enterprise/store/id/NodeIdReuseStressIT.java:132]: yeah i know, TC should be able to survive that right?

Comment (burqen) [reply enterprise/kernel/src/test/java/org/neo4j/kernel/impl/enterprise/store/id/NodeIdReuseStressIT.java:132]: Yeah, that's not a problem. It'll only make it slightly harder to parse, which is not a big deal.

**Review by burqen — APPROVED** [review]

**Pull request merged**